composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "yiisoft/yii2-queue",
+    "name": "dimanzver/yii2-queue",
```

Comment:
```suggestion
    "name": "yiisoft/yii2-queue",
```

Reply:
For handling of stopsignal in docker container. For example, in Kubernetes on deploy